bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -12,6 +12,16 @@
  * (I didn't want to add these to dependencies just for this script since these are kinda heavy ones)
  * https://www.npmjs.com/package/capture-website
  * https://www.npmjs.com/package/sharp
+ *
+ * Ensure browser is installed by removing `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true` in `package.json` and run `npm ci`
+ * (maybe there's easier way?)
```

Comment:
Running `npm ci` by itself won't have the env vars from package.json so should work on its own.

I believe just `npm i puppeteer` should work. 

Either of these assume you haven't exported the var in your local environment.